FrontEnd.cpp

```diff
@@ -52,15 +52,17 @@ using namespace Eigen;
 StaticFusion::StaticFusion(unsigned int res_factor)
 {
     //Resolutions and levels
-    rows = 480/res_factor;
-    cols = 640/res_factor;
-    fovh = M_PI*62.5/180.0;
-    fovv = M_PI*48.5/180.0;
-    width = 640/res_factor;
-    height = 480/res_factor;
+    width = Resolution::getInstance().width();
+    height = Resolution::getInstance().height();
+    cols = width;
+    rows = height;
+    cx = Intrinsics::getInstance().cx();
+    cy = Intrinsics::getInstance().cy();
+    fx = Intrinsics::getInstance().fx();
+    fy = Intrinsics::getInstance().fy();
+    fovh = 2 * atan2(width,2 * fx);
+    fovv = 2 * atan2(height,2 * fy);
```

Comment:
Since we are not using a parameter in the constructor, it is not obvious how this is going to be used. Use one of the demo programs to give an example how the `Resolution` and `Intrinsics` is going to be used.